docker-compose.yml

```diff
@@ -245,8 +251,10 @@ services:
     volumes: &conda-volumes
       - .:/arrow:delegated
       - ${DOCKER_VOLUME_PREFIX}conda-ccache:/ccache:delegated
-    command:
-      ["/arrow/ci/scripts/cpp_build.sh /arrow /build &&
+    command: &conda-cpp-command
+      ["/arrow/ci/scripts/install_sccache.sh unknown-linux-musl &&
```

Comment:
I'm curious: why "unknown-linux-musl"? Does our conda image really use musl?

Reply:
Also, can the `install_sccache.sh` call instead be moved to the dockerfile?

Reply:
> There are prebuilt x86-64 binaries available for Windows, **Linux (a portable binary compiled against musl),** and macOS 

I haven't moved the installation to the dockerfiles for ease of testing but we can do that.